src/components/switch/switch.scss

```diff
@@ -18,8 +18,7 @@
 }
```

Comment:
> fix(switch): make the structure more like the latest MDC

I'd say this is a refactor. Sure, it could theoretically affect a consumer who's relying on the internal html structure of the component to look a certain way, but they shouldn't be doing that anyway.

I can rename the commit, if you'd like me to?

Reply:
correct! I'll rename it